src/pipx/interpreter.py

```diff
@@ -0,0 +1,60 @@
+import shutil
+import subprocess
+import sys
+
+from pipx.constants import WINDOWS
+from pipx.util import PipxError
+
+
+def has_venv() -> bool:
+    try:
+        import venv  # noqa
+
+        return True
+    except ImportError:
+        return False
+
+
+# If we are running under the Windows embeddable distribution,
+# venv isn't available (and we probably don't want to use the
+# embeddable distribution as our applications' base Python anyway)
+# so we try to locate the system Python and use that instead.
+
+# This code was copied from https://github.com/uranusjr/pipx-standalone
+# which uses this technique to build a completely standalone pipx
+# distribution
```

Comment:
Do these comment blocks belong inside the function below instead?

Reply:
It was intentional to put the comments before the function rather than inside it (I find it more readable that way). But I can reword it a bit, I guess. Maybe swap the paragraphs, and say "The following code was copied from ... which uses the same technique..."?

Reply:
Yeah, the paragraphs seem more natural to me if swapped.

Reply:
Done.